### PR TITLE
Bring some inventor feats up to date with automation

### PR DESCRIPTION
**Pull request by LebombJames**
-Use `@Damage` in Explode
-Consolidate Deep Freeze effect (and link in the feat, it never was in the first place)
-Link to repair in Haphazard Repair

### CODE REVIEW

**Comment by TikaelSol** [comment]
If we're deleting effects (yay!) then we should migrate away old links. Or keep the effects and remove the links from the system, adding them to the larger tracking condense effects issue to be migrated away later.

**Comment by LebombJames** [comment]
There are no old links to Deep Freeze, it was never linked in the feat in the first place

**Comment by TikaelSol** [comment]
Good enough for me then

File: packs/feat-effects/effect-deep-freeze.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "deep-freeze:degree-of-success:critical-failure"
+                ],
+                "selector": "speed",
+                "slug": "deep-freeze-critical-failure",
+                "type": "circumstance",
+                "value": -15
+            },
```
Comment (TikaelSol) [inline packs/feat-effects/effect-deep-freeze.json:68]:
```suggestion
```

We can do this with one flat modifier. Just make the values of the choices `5`, `10`, and `15` then do `-@item.flags.pf2e.rulesSelections.speedPenalty` (along with changing the flag to make sense)

The grant will need to change its predicate to `deep-freeze-penalty:15` but it's the nicer way to do the penalty I think.

File: packs/feat-effects/effect-deep-freeze.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                "alterations": [
+                    {
+                        "mode": "override",
+                        "property": "badge-value",
+                        "value": 1
+                    }
+                ],
+                "key": "GrantItem",
+                "predicate": [
+                    "deep-freeze:degree-of-success:critical-failure"
```
Comment (TikaelSol) [inline packs/feat-effects/effect-deep-freeze.json:79]:
```suggestion
                    "deep-freeze-penalty:15"
```

File: packs/feat-effects/effect-deep-freeze.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "deep-freeze:degree-of-success:success"
+                ],
+                "selector": "speed",
+                "slug": "deep-freeze-success",
+                "type": "circumstance",
+                "value": -5
+            },
```
Comment (TikaelSol) [inline packs/feat-effects/effect-deep-freeze.json:48]:
```suggestion
            {
                "key": "FlatModifier",
                "selector": "all-speeds",
                "type": "circumstance",
                "value": "-@item.flags.pf2e.rulesSelections.speedPenalty"
            },
```

We prefer this selector to remove any ambiguity when users look.

File: packs/feat-effects/effect-deep-freeze.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
+                        "value": "critical-failure"
+                    }
+                ],
+                "flag": "deepFreezeDegreeOfSuccess",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.UI.RuleElements.ChoiceSet.Prompt",
+                "rollOption": "deep-freeze:degree-of-success"
+            },
```
Comment (TikaelSol) [inline packs/feat-effects/effect-deep-freeze.json:38]:
```suggestion
            {
                "choices": [
                    {
                        "label": "PF2E.Check.Result.Degree.Check.success",
                        "value": "5"
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.failure",
                        "value": "10"
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
                        "value": "15"
                    }
                ],
                "flag": "speedPenalty",
                "key": "ChoiceSet",
                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess",
                "rollOption": "deep-freeze-penalty"
            },
```

Comment (LebombJames) [reply packs/feat-effects/effect-deep-freeze.json]: Should the speed penalty values be strings or numbers?

Comment (TikaelSol) [reply packs/feat-effects/effect-deep-freeze.json]: Numbers actually, I was thinking of suboptions which don't allow numbers but choices do.

File: packs/feat-effects/effect-deep-freeze.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                        "value": "5"
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.failure",
+                        "value": "10"
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
+                        "value": "15"
+                    }
```
Comment (TikaelSol) [inline packs/feat-effects/effect-deep-freeze.json:32]:
```suggestion
                    {
                        "label": "PF2E.Check.Result.Degree.Check.success",
                        "value": 5
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.failure",
                        "value": 10
                    },
                    {
                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
                        "value": 15
                    }
```

**Pull request merged**